zerver/tests/test_openapi.py

```diff
@@ -171,7 +171,7 @@ class OpenAPIArgumentsTest(ZulipTestCase):
         '/users/me/presence',
         '/users/me/alert_words',
         '/users/me/status',
-
+        '/users/{user_id}/subscription/{stream_id}',
```

Comment:
I'd prefer that we take the opportunity to fully add this to `zerver/openapi/zulip.yaml` including with `call_endpoint` style documentation, given that you're working on this :)